libs/ngx-dsfr/radio/src/lib/radio.component.ts

```diff
@@ -12,8 +12,14 @@ export const EMPTY_LEGEND_ERROR: string =
   'You MUST provide a legend for these radio buttons 😡 !!!';
 export const EMPTY_NAME_ERROR: string =
   'You MUST provide a name for these radio buttons 😡 !!!';
+export const EMPTY_FIELDSET_ID_ERROR: string =
+  'You MUST provide a value for the fieldsetId attribute 😡 !!!';
 export const EMPTY_ITEMS_ERROR: string =
   'You MUST provide a value for the items attribute 😡 !!!';
+export const EMPTY_FAILURE_ID_OR_MESSAGE_ERROR: string =
+  'You MUST provide a value for the failureId and failureMessage attributes when hasFailed is true 😡 !!!';
+export const EMPTY_SUCCESS_ID_OR_MESSAGE_ERROR: string =
+  'You MUST provide a value for the successId and successMessage attributes when hasSucceeded is true 😡 !!!';
```

Comment:
Don't forget to change these 2 error messages with the removal of `failureId` and `successId`